packages/open-truss/src/configuration/engine-v1/RenderConfig.tsx

```diff
@@ -94,14 +100,16 @@ function createSignals(signalsConfig: SignalsV1, validate: boolean): Signals {
   const signals: Signals = {}
   if (signalsConfig === undefined) return signals
   Object.entries(signalsConfig).forEach(([name, val]) => {
-    const signal = SIGNALS[val]?.signal
-    if (validate && signal === undefined) {
-      const err = `Signal type ${String(
-        val,
-      )} is unknown. Please check value of ${String(name)} Signal`
-      throw new Error(err)
+    let signal: SignalsZodType | undefined
```

Comment:
@kmcq unfortunately I had to keep the `undefined` here because this variable can be undefined if none of the conditions below matched.